ccs.wdl

```diff
@@ -77,6 +80,8 @@ task CCS {
         minReadQuality: {description: "Minimum predicted accuracy in [0, 1].", category: "common"}
         logLevel: {description: "Set log level. Valid choices: (TRACE, DEBUG, INFO, WARN, FATAL).", category: "advanced"}
         subreadsFile: {description: "Subreads input file.", category: "required"}
+        subreadsIndexFile: {description: "Index for the subreads input file.", category: "required"}
```

Comment:
Is the index required or optional? The input is now optional, so users could leave it out.

Reply:
It it optional, unless you want to use scattering.

Reply:
I think the category should be `"advanced"` in that case right @DavyCats? Maybe mention it is required when you want to do scattering in the `description`.